templates/admin/adherent/list_mandate.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    {% if object.mandate %}
```

Comment:
what about moving this `if block` in one other partial template like `_mandate.html.twig` and reuse it in `show_mandate.html.twig` for avoiding the code duplication?